ExilenceNextApp/src/components/settings-tabs/text-input-setting/TextInputSetting.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { createStyles, makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(theme =>
+  createStyles({
+    root: {
+      marginTop: theme.spacing(1),
+      marginBottom: theme.spacing(1),
+      '&::after': {
+        content: 'min !important'
```

Comment:
`content: 'min !important'` will add a string after the content that says `min !important` - is this intended? Normally we use `content: 'min' !important;` like so but even then the `!important` part is probably unnecessary.

Reply:
I think this is copied from another setting component we use, I suggest leave it and refactor all of these together at a later point